.env.example

```
@@ -0,0 +1,49 @@
+OPENAI_TEMPERATURE=0.1
+
+# -----------------------------------------------------------------------------
+# Server Configuration
+# -----------------------------------------------------------------------------
+PORT=8080
+
+# -----------------------------------------------------------------------------
+# Application Configuration
+# -----------------------------------------------------------------------------
+APP_NAME=GraphbitReviewAgent
+DEBUG=False  # Must be false in production
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)